apis/v1alpha1/ack-generate-metadata.yaml

```diff
@@ -1,8 +1,8 @@
 ack_generate_info:
-  build_date: "2022-09-06T12:47:50Z"
-  build_hash: 585f06bbd6d4cc1b738acb85901e7a009bf452c7
-  go_version: go1.17.5
-  version: v0.20.0
+  build_date: "2022-09-20T05:27:13Z"
+  build_hash: 1da44f4be322eea156ed23a86e33c29da5cede9c
+  go_version: go1.19.1
+  version: v0.20.1-2-g1da44f4-dirty
```

Comment:
Please update make the upgrade of runtime and code generator to `v0.20.1` in a seperate PR. The current runtime still 2 versions behijnd

Reply:
thanks done